requirements-testing.txt

```diff
@@ -2,3 +2,4 @@ pytest
 pytest-cov
 ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow NWBFile load at all timess
 parameterized>=0.8.1
+pre-commit >= 2.20.0
```

Comment:
Why is this needed?

Reply:
It is not. I added it because it was a development tool that we use but don't have anywhere else when I was installing. It slipped in this PR by mistake though. Feel free to remove it.

Reply:
We harness the pre-commit bot at the end of the day for this, but people are always free to install and use locally (but not a part of package or even required of test/dev installations)